### ISSUE TITLE
Add delete handler for Informer in newPodsApi (sources/pods.go)

### ISSUE THREAD

**Issue by mwielgus** [kubernetes/heapster#514]
Subtask of: #475
Depends on: #508, #513 

So that pod deletions handled by podApi are propagated to Model implementation.

**Comment by mwielgus** [comment]
This can be postponed.

**Comment by piosz** [comment]
Obsolete.
